core/rio/ntriples/src/main/java/org/eclipse/rdf4j/rio/ntriples/NTriplesUtil.java

```diff
@@ -67,7 +62,7 @@ else if (nTriplesValue.startsWith("\"")) {
 	 *         If the supplied resource could not be parsed correctly.
 	 */
 	public static Resource parseResource(String nTriplesResource, ValueFactory valueFactory)
-		throws IllegalArgumentException
+			throws IllegalArgumentException
```

Comment:
Have you applied the RDF4J code formatting settings? It looks like the indentation is off.